src/core.js

```diff
@@ -214,10 +215,10 @@ export class JSONEditor {
     return JSONEditor.defaults.editors[classname]
   }
 
-  createEditor (editorClass, options) {
+  createEditor (editorClass, options, recursion = 0) {
```

Comment:
please call this argument something like `recursionCounter` to clear its purpose